content/en/docs/tasks/administer-cluster/migrating-from-dockershim/_index.md

```diff
@@ -0,0 +1,24 @@
+---
+title: "Migrating from dockershim to other container runtimes"
+weight: 10
+content_type: task 
+---
+
+<!-- overview -->
+
+This section presents information you need to know when migrating from
+dockershim to other container runtimes.
+
+Since the announcement of [dockershim deprecation](/blog/2020/12/08/kubernetes-1-20-release-announcement/#dockershim-deprecation)
+in Kubernetes 1.20, there were questions on how this will affect various workloads and Kubernetes
+installations. You can find these links useful to understand the problem better:
+
```

Comment:
nit: Is there one blog that refers to the other blogs? I think one link could bring in the other articles.

Reply:
agree. Reworded and only left the FAQ